velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -262,7 +268,12 @@ void Writer::close() {
 }
 
 void Writer::abort() {
-  VELOX_NYI("abort function for Parquet writer is not supported yet");
+  stream_->abort();
+  if (arrowContext_->writer) {
+    arrowContext_->writer.reset();
+  }
+  arrowContext_->stagingChunks.clear();
```

Comment:
Just `arrowContext_.reset()` will reset all its members also?

Reply:
Got it thanks.